### ISSUE TITLE
Bilder in der Datenbank speichern

### ISSUE THREAD

**Issue by phxql** [qaware/wir-vs-virus#220]
Wenn wir unsere Software irgendwann mal containerisieren / kubernetisieren, ist's nix mehr mit lokalem Dateisystem. Dann sollten wir die Bilder in der Datenbank speichern.

**Comment by defaude** [comment]
Wäre da ein S3 Bucket oder so nicht ebenfalls denkbar? Auch containerisiert muss man auf so schicken Cloud-Kram doch zugreifen können.

**Comment by phxql** [comment]
Jo, aber da baut man dann halt gegen ne Cloud und hat vendor lock-in.
